### PR TITLE
Add calendar section to documentation. Explanations and example is pr…

### PR DESCRIPTION
**Pull request by enter360**
…ovided.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

## Summary by CodeRabbit

- **New Features**
	- Introduced a "Calendar" section in the Ridwell integration documentation, detailing attributes and providing an example JSON message for clarity on calendar functionality related to waste recycling pickups.

### CODE REVIEW

**Comment by enter360** [comment]
update branch

File: source/_integrations/ridwell.markdown [hunk 1]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
 
 {% include integrations/config_flow.md %}
 
+## Calendar
+
+The calendar sensor contains a `calendar` state attribute, which contains:
+
+- The name of the Ridwell Calendar
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
- The name of the Ridwell Calendar.
```

File: source/_integrations/ridwell.markdown [hunk 2]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
...
+- Boolean for all day occurrence of event
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
- If the event is an all-day event (Boolean).
```

File: source/_integrations/ridwell.markdown [hunk 3]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
...
+- The start time of the calendar event. Starts at midnight
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
- The start time of the calendar event. Starts at midnight.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Tiny tweaks

File: source/_integrations/ridwell.markdown [hunk 4]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
...
+
+- The name of the Ridwell Calendar.
+- If the event is an all-day event (Boolean).
+- The start time of the calendar event. Starts at midnight.
+- The end time of the event. Ends at midnight.
+- The location of the event.
+- The description of the items being picked up.
+- Friendly name of Calendar Event.
+
+For Example:
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
For example:
```

File: source/_integrations/ridwell.markdown [hunk 5]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
...
+
+The calendar sensor contains a `calendar` state attribute, which contains:
+
+- The name of the Ridwell Calendar.
+- If the event is an all-day event (Boolean).
+- The start time of the calendar event. Starts at midnight.
+- The end time of the event. Ends at midnight.
+- The location of the event.
+- The description of the items being picked up.
+- Friendly name of Calendar Event.
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
- Friendly name of the calendar event.
```

File: source/_integrations/ridwell.markdown [hunk 6]
```diff
@@ -21,6 +21,30 @@ The Ridwell integration allows users to track waste recycling pickups scheduled
...
+- If the event is an all-day event (Boolean).
+- The start time of the calendar event. Starts at midnight.
+- The end time of the event. Ends at midnight.
+- The location of the event.
+- The description of the items being picked up.
+- Friendly name of the calendar event.
+
+For example:
+
+```json
```
Comment (c0ffeeca7) [inline source/_integrations/ridwell.markdown]:
```suggestion
```yaml
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @enter360 :+1:

**Pull request merged**